util/reggen/multi_register.py

```diff
@@ -89,13 +91,15 @@ class MultiRegister(RegBase):
                    of reg divide evenly into a whole number of concrete
                    registers).
 
-      needs_qe     This is true iff at least one of the pseudo-registers in the
+      needs_qe     This is true if at least one of the pseudo-registers in the
                    multi-register needs a q-enable signal.
     """
 
     def __init__(self,
                  name: str,
                  offset: int,
+                 count: int,
+                 stride: int,
```

Comment:
This looks like it's defining new fields for multireg. That may raise some flags for further discussion.

Reply:
Oh, I see what you mean. `count` was even documented but not implemented, and `addrsep` got removed in https://github.com/lowRISC/opentitan/pull/27289/commits/96c5ac7de4fbfd50ef2af34df0285ac3c6c3d70a, haha.